src/io/flutter/pub/PubRoot.java

```diff
@@ -56,6 +56,20 @@ private PubRoot(@NotNull VirtualFile root, @NotNull VirtualFile pubspec, @Nullab
     this.lib = lib;
   }
 
+  /**
+   * Returns the first containing pub root for a given file.
```

Comment:
I don't understand this comment. Is a word missing, perhaps?

Reply:
Yes.  Very awkward.  Will fix.  Thanks!